src/sentry/integrations/jira/views/issue.py

```diff
@@ -26,17 +25,6 @@ def accum(tot, item):
 class JiraIssueHookView(JiraBaseHook):
     html_file = "sentry/integrations/jira-issue.html"
 
-    def set_badge(self, integration, issue_key, group_link_num):
```

Comment:
I'd prefer this stay named `issue_hook.py` because there are also Jira issues, and the issue hook is a specific thing so this naming is confusing. I'm also not sure why `set_badge` is being moved out of this file, it's the only place it'll ever be called because it's specific to issue hooks.

Reply:
Renaming the file now.
I'm moving `set_badge()` into a file with `get_assignee_email()` and `handle_assignee_change()` to abstract away client configuration and API calls from the view layer.